app/assets/stylesheets/components/cards/prompt.scss

```diff
@@ -0,0 +1,91 @@
+@use "govuk-frontend/dist/govuk" as govuk;
+@use "../../colours";
+
+.prompt {
+  margin: 45px 0 25px;
+  padding-left: 20px;
+  padding-right: 20px;
+  position: relative;
+  background-color: #f3f2f1;
+  border-top: #28a197 solid 10px;
+}
+.prompt .svg-inline--fa {
+  background-color: #28a197;
+  color: #ffffff;
+  padding: 10px !important;
+  position: absolute;
+  display: inline-block;
+  width: 32px;
+  top: -30px;
+  left: 50%;
+  margin-left: -32px;
+
+  @media (min-width: 48.0625em) {
+    position: relative;
+    left: 5px;
+    top:-20px;
+    padding: 20px !important;
+  }
+
+}
+.prompt__container{
+  padding: 40px 10px 15px 10px;
+
+  @media (min-width: 48.0625em) {
+    padding: 20px 50px 50px 80px;
+  }
+
+  h2.govuk-heading-l{
+    padding-left: 0px;
+  }
+}
+
+.prompt .govuk-grid-column-one-quarter {
+  margin-right: -120px;
+
+  @media (min-width: 48.0625em) {
```

Comment:
We should use the gov breakpoints where possible, helps everything scale at the same points 

https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-media-query